### ISSUE TITLE
Switching off does not work with MegaBoom 3 and firmware 1.6.45

### ISSUE THREAD

**Issue by splintering** [Shingyx/BoomSwitch#8]
Switching on my MegaBook 3 works properly with the app or the widget, but switching off the speaker does not work. The following messages appear after a couple of seconds each:

"Connecting to Megaboom 3"
"Retrying connection with Megaboom 3"
"Connection failed. Is Megaboom 3 in range?"

Switching off works with the official UE app (v. 7.6.0.277) though.

MegaBoom 3 firmware v 1.6.45
BOOM Switch v 1.3
Android 11 / Samsung Galaxy A51

**Comment by Shingyx** [comment]
Hi there, thanks for reporting this issue.

Being able to power the speaker on, but not being able to power the speaker _off_, sounds very similar to a previous issue - https://github.com/Shingyx/BoomSwitch/issues/5

The summary was that sometimes, having both the official app _and_ BOOM Switch running at the same time causes some kind of conflict which "disables" the switching off functionality using Bluetooth LE (Low Energy). Switching the speaker off using the official app still works in this situation because the official app does not use Bluetooth LE for switching the speaker _off_.

I admit the error messages you saw need improvement. The error in this case would likely be "failed to connect to the speaker using Bluetooth LE", rather than "the speaker is not in range". I'll find some better wording at some stage.

Can you please try either force-stopping the Ultimate Ears BOOM app ([I found a guide here](https://support.pix4d.com/hc/en-us/articles/209626486-How-to-force-quit-an-App-PIX4Dcapture)), or uninstalling it, and then try using BOOM Switch again?

Please let me know how you go. Cheers.

**Comment by splintering** [comment]
I opted for uninstalling the official app. Switching off the speaker works as expected then, so it seems you are right. I somehow even have the feeling that switching on goes quicker then, but I made no real comparison.

I might leave it that way, because the only other thing I would use the official app for is firmware upgrade, and for that I can install and uninstall it from time to time. Having the Boom switch widget is way more handy than opening the official app and using that for switching on/off (especially at night, as it has a white background and no dark mode).

Maybe that conflict between the apps is something that can be put into the readme.

Thanks for the prompt response btw :-)
